doc/contributing.md

```diff
@@ -50,6 +50,27 @@ top of the GitHub repository and click *New Issue*.
 * After submitting your bug report, try to answer any follow up questions about the bug
   as best as you can.
 
+#### Reporting upstream bugs
+
+If you are aware that a bug is caused by an upstream GMT issue rather than a
+PyGMT-specific issue, you can optionally take the following steps to help resolve
+the problem:
+
+* Add the line `pygmt.config(GMT_VERBOSE='d')` after your import statements, which
+  will report the equivalent GMT commands as one of the debug messages.
+* Either append all messages from running your script to your GitHub issue, or
+  filter the messages to include only the GMT-equivalent commands using a command
+  such as:
+
+      python <test>.py 2>&1 | grep GMT_Call_Command | awk -F': ' '{print "gmt", $3}'
```

Comment:
What about this command? It doesn't call `grep`.
```suggestion
      python test.py 2>&1 | awk -F': ' '$2=="GMT_Call_Command string" {print "gmt", $3}'
```

Reply:
BTW, this command works for both Linux (GNU awk) and macOS (BSD awk).

Reply:
Do we want to have an equivalent command for Windows users? :slightly_smiling_face:

Reply:
Does Windows provide any built-in tools to do tasks like awk?

Reply:
A quick search yields this https://superuser.com/questions/524822/awk-equivalent-functionality-on-windows/524836#524836, but not sure if it works with stderr/stdout from GMT.

Reply:
I don't really think we need to spend time on this, since it only applies to the subset of people who are voluntarily providing additional information about upstream bugs without WSL or Git for Windows installed. They can always post the full error message or we could extract the equivalent commands on a *nix system.